test/test_namedtensor.py

```diff
@@ -988,6 +988,13 @@ def test_ops(op):
         test_ops(torch.cummax)
         test_ops(torch.cummin)
 
+    def test_logcumsumexp(self):
+        for device in torch.testing.get_all_device_types():
+            names = ('N', 'D')
+            tensor = torch.rand(2, 3, names=names)
+            result = torch.logcumsumexp(tensor, 0)
```

Comment:
I would have expected passing the dimension name here ('N' or 'D'), not an integer.